src/components/tiger/tiger.tsx

```diff
@@ -80,18 +81,6 @@ const Tiger = () => {
         setLoading(true);
 
         let args = ["-l", values.reportFile];
-
-        CommandHelper.runCommandWithPkexec("tiger", args, handleProcessData, handleProcessTermination)
```

Comment:
I only wanted you to remove one line: setLoading(false). If you remove CommandHelper completely, then the tool will not run. You need to reinstate it.